.gitignore

```diff
@@ -0,0 +1 @@
+.idea/*
```

Comment:
it's better to put this into your local gitignore in `~/.gitignore` instead of adding it to each repository you work on.